doc/optimizers.md

```diff
@@ -195,6 +195,53 @@ optimizer.Optimize(f, coordinates);
  * [Adam: A Method for Stochastic Optimization](http://arxiv.org/abs/1412.6980) (see section 7)
  * [Differentiable separable functions](#differentiable-separable-functions)
 
+## Padam
```

Comment:
If you don't mind I think we should move this to alphabetical ordering like the other optimizers.